odoo/tools/config.py

```diff
@@ -488,10 +489,14 @@ def die(cond, msg):
                 self._normalize(x)
                 for x in self.options['addons_path'].split(','))
 
-        self.options['upgrades_paths'] = (
-            ",".join(self._normalize(x)
-                for x in self.options['upgrades_paths'].split(','))
-            if self.options['upgrades_paths']
+        self.options["upgrades_paths"] = (
+            ",".join(
+                np
+                for p in self.options["upgrades_paths"].split(",")
+                for np in [self._normalize(p)]
+                if np
```

Comment:
Why?

Reply:
Trailing comma?  Seriously?

Reply:
It can be removed.
Then, also remove https://github.com/odoo/odoo/pull/43681/files#diff-3813bc998e290bd756b27aef89662e82R564-R565